src/TypeUtils.ts

```diff
@@ -0,0 +1,57 @@
+import * as _ from 'lodash'
+
+const equality = (from:string, to:string) => ['Equals', from, to]
+
+const TypeUtils = {
+	nativeFactTypeTemplates: {
+		equality: {
+			  'is equal to': equality
```

Comment:
Style again, but any separator/operator on the end of a line break:
```typescript
{
    'is equal to': equality,
    'equals': equality,
}
```
Makes it a lot easier to read IMHO. We've made the `,` on the end of an object optional, but as @Page- once pointed out, it makes it a lot easier to add new values later.